python/plugins/db_manager/dlg_sql_window.py

```diff
@@ -595,8 +595,14 @@ def createView(self):
     def _getSqlQuery(self):
```

Comment:
I think this function is used to store queries in preset or in file. They are many usages. So we will loose comments in saved file.

Reply:
@Gustry ok, I'll move it down into the executor

Reply:
@Gustry, looking better I thing you are wrong, the original behavior was altering the query by replacing newlines with spaces, and that was the source of the issue: https://github.com/qgis/QGIS/pull/9180/files#diff-9020444f364248acb9ad107f6fb38ba0L598

Reply:
`getSqlQuery` is used when we store the query as file: https://github.com/qgis/QGIS/blob/697237227d242c674940961b3374e9e55d928a0a/python/plugins/db_manager/dlg_sql_window.py#L234 so it means that the content of the file is not the same? (especially with this PR)

Sorry if I didn't understand. I will look later.